cli/args/flags.rs

```diff
@@ -4192,8 +4207,8 @@ impl Iterator for UnstableArgsIter {
         UnstableArgsConfig::ResolutionOnly | UnstableArgsConfig::ResolutionAndRuntime => Some("true")
       })
       .help_heading(UNSTABLE_HEADING)
-    } else if self.idx > 3 {
-      let granular_flag = crate::UNSTABLE_GRANULAR_FLAGS.get(self.idx - 4)?;
+    } else if self.idx > 4 {
+      let granular_flag = crate::UNSTABLE_GRANULAR_FLAGS.get(self.idx - 5)?;
```

Comment:
Would be better if the code didn't require changing numbers like this.

Reply:
CC @crowlKats

Reply:
this could just be an `else` case instead

Reply:
There is already an `else`. Why do we do this instead of just constructing a vector?

Reply:
I removed this code in https://github.com/denoland/deno/pull/26149/commits/fdbb7a42a5658536b1dcf08af32e69a2c9e07910